inputParameterRanges.js

```diff
@@ -0,0 +1,142 @@
+const TemperatureRange = {
+  min: 0,
+  max: 45,
+  thresholdLow: 2.25,
+  thresholdHigh: 42.75,
+};
+const SOCRange = { min: 20, max: 80, thresholdLow: 24, thresholdHigh: 76 };
+
+const ChargeRateRange = {
+  min: 0,
+  max: 0.8,
+  thresholdLow: 0.04,
+  thresholdHigh: 0.76,
+};
+const languageCode = 1; // 1.English// 2. German
```

Comment:
If the `messageHashMap` is indexed using strings, why have numbers for the language? Keep strings itself, then you dont need to lookup this comment whenever you encounter a language.
For example, you could have 'eng_temperature_low' and 'ger_temperature_low' as the key instead of '1Temperature1' etc